Nanoleaf-wpf/Views/MainWindows/MainWindow.xaml

```diff
@@ -14,28 +14,28 @@
             <Button  Grid.Row="0" VerticalAlignment="Top" Height="35" Background="{x:Null}" BorderBrush="{x:Null}" Click="AddSchedule_Click" Margin="0,51,10,0" HorizontalAlignment="Right" Width="35">
                 <Image Source="../../Icons/appbar.add.png"/>
             </Button>
-            <StackPanel Height="Auto" Margin="10,86,10,0" VerticalAlignment="Stretch" x:Name="ScheduleList">
+            <StackPanel Height="Auto" Margin="10,86,10,104.333" VerticalAlignment="Stretch" x:Name="ScheduleList">
                 <Border BorderBrush="White" BorderThickness="0 1 1 0" Height="Auto"/>
             </StackPanel>
             <Label Content="{x:Static mainWindows:Resources.ActiveDevice}" ContentStringFormat="{}{0}:" HorizontalAlignment="Left" VerticalAlignment="Top" FontSize="20" Foreground="White" Margin="10,0,0,0"/>
             <ComboBox x:Name="DevicesDropdown" Margin="0,10,10,0" VerticalAlignment="Top" SelectedItem="{Binding SelectedDevice}" ItemsSource="{Binding DeviceNames}" HorizontalAlignment="Right" Width="160"/>
+            <Button Content="{x:Static mainWindows:Resources.ReloadEffects}" HorizontalAlignment="Left" Margin="10,0,0,10" Width="118" Background="{StaticResource NanoleafGreen}" Foreground="White" BorderBrush="{x:Null}" Click="Reload_Click" Height="27" VerticalAlignment="Bottom"/>
+            <Button Content="{x:Static mainWindows:Resources.RemoveDevice}" Margin="0,0,10,10" Background="{StaticResource NanoleafGreen}" Foreground="White" BorderBrush="{x:Null}" Click="RemoveDevice_Click" HorizontalAlignment="Right" Width="135" Height="27" VerticalAlignment="Bottom"/>
+            <Label Content="{x:Static mainWindows:Resources.DeviceActions}" ContentStringFormat="{}{0}:" HorizontalAlignment="Left" FontSize="15" Foreground="White" Margin="10,0,0,74" Height="30" VerticalAlignment="Bottom"/>
+            <Button Content="{x:Static mainWindows:Resources.PercentageProfile}" HorizontalAlignment="Left" Margin="10,0,0,42" Width="118" Background="{StaticResource NanoleafGreen}" Foreground="White" BorderBrush="{x:Null}" Click="PercentageProfile_Click" Height="27" VerticalAlignment="Bottom"/>
         </Grid>
         <Grid>
             <Button Content="{x:Static mainWindows:Resources.Options}" HorizontalAlignment="Left" Margin="10,10,0,0" VerticalAlignment="Top" Width="100" Height="27" Background="{StaticResource NanoleafGreen}" Foreground="White" BorderBrush="{x:Null}" Click="Options_Click"/>
-            <Button Content="{x:Static mainWindows:Resources.ResetAllSettings}" HorizontalAlignment="Right" Margin="0,0,10,10" VerticalAlignment="Bottom" Width="120" Height="27" Background="{StaticResource NanoleafGreen}" Foreground="White" BorderBrush="{x:Null}" Click="Reset_Click"/>
+            <Button Content="{x:Static mainWindows:Resources.ResetAllSettings}" Margin="240,10,102,0" Background="{StaticResource NanoleafGreen}" Foreground="White" BorderBrush="{x:Null}" Click="Reset_Click" Height="27" VerticalAlignment="Top"/>
```

Comment:
3 Alignments is most likely wrong, try to keep it on 2 for scalability, need to test this.

Reply:
You are right, fixed